### DISCUSSION TITLE
Weber iGrill 2

### DISCUSSION THREAD

**Discussion by Marcoletto** [custom-components/ble_monitor#1118] [Ideas]
Hi guys,
is it possible to insert the Weber iGrill 2 in HA?

How I can pairing with HA with his integration?

Many thanks,
Bye

**Comment by Ernst79** [comment]
We can try. Check https://custom-components.github.io/ble_monitor/sensor_request to get some info I need

**Reply by Marcoletto** [reply]
Hi Ernst, unfortunately i?m not skilled to follow this procedure to give you the right data.
I hope that other peoples with iGrill2 are able to do this.

Thank you in advance
